features/old/forums/privacy.feature

```diff
@@ -9,14 +9,32 @@ Feature: Forum privacy
     And provider "foo.3scale.localhost" has "forum" enabled
     And the current domain is foo.3scale.localhost
 
-  Scenario: Private forum requires logged in user
-    Given the forum of provider "foo.3scale.localhost" is private
-    When I am not logged in
+  Rule: Forum is enabled
+    Background:
+      And I have rolling updates "forum" enabled
+
+    Scenario: Private forum requires logged in user
+      Given the forum of provider "foo.3scale.localhost" is private
+      When I am not logged in
       And I go to the forum page
-    Then I should be on the login page
+      Then I should be on the login page
 
-  Scenario: Public forum does not require a logged in user
-    Given the forum of provider "foo.3scale.localhost" is public
-    When I am not logged in
+    Scenario: Public forum does not require a logged in user
+      Given the forum of provider "foo.3scale.localhost" is public
+      When I am not logged in
       And I go to the forum page
-    Then I should be on the forum page
+      Then I should be on the forum page
+
+  Rule: Forum is disabled
```

Comment:
btw why should we switch to this syntax?

Reply:
AFAIK this is the only way to branch a feature file with different Background blocks. It's just a way to override previews Background with minor changes, in this case, disabling forums.